src/test/java/pl/piekoszek/gorskimatches/controllers/ImageControllerTest.java

```diff
@@ -0,0 +1,35 @@
+package pl.piekoszek.gorskimatches.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class ImageControllerTest {
+
+    String equation = "4+4=8";
```

Comment:
looks like constant, so you can make it constant by adding `static final`, also `private` wouldn't hurt